source/_docs/z-wave/adding.markdown

```diff
@@ -3,6 +3,10 @@ title: "Z-Wave Devices - Adding and Removing"
 description: "How to add and remove Z-Wave devices."
 ---
 
+**Recommendation before adding any devices:**
```

Comment:
If you need to emphasize it like a header, make it a header.

```suggestion
## Recommendation before adding any devices
```

Reply:
will update in the next commit